### ISSUE TITLE
Bug: Script seems to strip metadata from downloaded assets

### ISSUE THREAD

**Issue by francispeixoto** [espiegel/dojo#3]

**Comment by espiegel** [comment]
What kind of metadata do you expect there to be on the downloaded asset? And what is the actual metadata?

**Comment by francispeixoto** [comment]
Mostly exif. The images I downloaded had their exif stripped, so all the dates tags were reset to the download date. Your script is actually part of a chain, the next step being to sync the files to google photos. No exif means that photos can't place the pictures properly in the timeline

**Comment by francispeixoto** [comment]
So looking into it more, I think the culprit is classdojo that strips the exif when the educator uploads a picture. I figure it's a feature to meet privacy requirements or something. I forked your repo and I'm going to look into modules too edit the exif. I figure I can reinject exif data from the content timestamps

**Comment by espiegel** [comment]
Makes sense that classdojo strips it out. 
Although you can only reinject the time that the photo was uploaded, not the time the photo was taken. But I guess that's better than nothing.

**Comment by francispeixoto** [comment]
Yeah it's not the most accurate timestamp, but so far with my daycare, they're uploading on the same day, so I'm not to worried. I can accept a few days discrepancy. I just can't have two year old photos bunched up with yesterday's.
